commands/start.go

```diff
@@ -54,6 +54,17 @@ func StartClients(ctx *cli.Context) (err error) {
 		return utils.Exit(fmt.Sprintf("❌  There was an error while preparing %s flags: %v", consensusClient.Name(), err), 1)
 	}
 
+	if ctx.Bool(flags.CheckpointSyncFlag) {
+		log.Info("⚙️   Checkpoint sync feature enabled")
+
+		checkpointURL := "https://checkpoints.mainnet.lukso.network/"
```

Comment:
what if its devote, and a user provides this flag, will it error to the user?

Reply:
or will it try to use the mainnet, which would not work.

Reply:
> what if its devote, and a user provides this flag, will it error to the user?

Currently devnet is disabled, so it wont get to this point

Reply:
@Wolmin We will need to secure it.

Reply:
on devote or any other future network please just log with (no error or exit, just skip the flag) with:
`"This network doesn't have a checkpoint sync setup"`

Reply:
please also make the URLs global vars,

Reply:
Solved. Secured and move checkpoints URLs to config vars.